config/notifiers.go

```diff
@@ -456,16 +457,18 @@ type OpsGenieConfig struct {
 
 	HTTPConfig *commoncfg.HTTPClientConfig `yaml:"http_config,omitempty" json:"http_config,omitempty"`
 
-	APIKey      Secret                    `yaml:"api_key,omitempty" json:"api_key,omitempty"`
-	APIURL      *URL                      `yaml:"api_url,omitempty" json:"api_url,omitempty"`
-	Message     string                    `yaml:"message,omitempty" json:"message,omitempty"`
-	Description string                    `yaml:"description,omitempty" json:"description,omitempty"`
-	Source      string                    `yaml:"source,omitempty" json:"source,omitempty"`
-	Details     map[string]string         `yaml:"details,omitempty" json:"details,omitempty"`
-	Responders  []OpsGenieConfigResponder `yaml:"responders,omitempty" json:"responders,omitempty"`
-	Tags        string                    `yaml:"tags,omitempty" json:"tags,omitempty"`
-	Note        string                    `yaml:"note,omitempty" json:"note,omitempty"`
-	Priority    string                    `yaml:"priority,omitempty" json:"priority,omitempty"`
+	APIKey           Secret                    `yaml:"api_key,omitempty" json:"api_key,omitempty"`
+	APIURL           *URL                      `yaml:"api_url,omitempty" json:"api_url,omitempty"`
+	Message          string                    `yaml:"message,omitempty" json:"message,omitempty"`
+	Description      string                    `yaml:"description,omitempty" json:"description,omitempty"`
+	Source           string                    `yaml:"source,omitempty" json:"source,omitempty"`
+	Details          map[string]string         `yaml:"details,omitempty" json:"details,omitempty"`
+	DetailsLabels    []string                  `yaml:"details_labels,omitempty" json:"details_labels,omitempty"`
```

Comment:
This seems redundant, you can already do this with the existing config.

Reply:
Yes but as mentioned in https://github.com/prometheus/alertmanager/issues/682#issuecomment-476138382 this would be a shorter form of it. Also I think the overhead isn't that much and it is more flexible than introducing a `detailsLabelsAll` parameter.

Reply:
I changed this, so that there is only a `details_use_common_labels` option.